vaadin-spring-tests/test-spring-security-fusion-jwt/src/main/java/com/vaadin/flow/spring/fusionsecurityjwt/config/SecurityConfiguration.java

```diff
@@ -47,21 +44,15 @@ protected void configure(HttpSecurity http) throws Exception {
             .sessionManagement()
                     .sessionCreationPolicy(SessionCreationPolicy.STATELESS);
 
-        setJwtSplitCookieAuthentication(http, "statelessapp", 3600,
-                JWSAlgorithm.HS256);
+        setJwtSplitCookieAuthentication(http,
```

Comment:
Would it be good to have another overloaded method that takes a string as a secret? so that I can call
```
setJwtSplitCookieAuthentication(http,
                "I72kIcB1UrUQVHVUAzgweE+BLc0bF8mLv9SmrgKsQAk=", "statelessapp");
```